operate/client/e2e-playwright/tests/batchMoveModification.spec.ts

```diff
@@ -118,11 +118,7 @@ test.describe('Process Instance Batch Modification', () => {
       modificationOperationEntry.getByRole('progressbar'),
     ).not.toBeVisible();
 
-    await modificationOperationEntry
-      .getByRole('link', {
-        name: `${NUM_SELECTED_PROCESS_INSTANCES} Instances`,
-      })
-      .click();
+    await modificationOperationEntry.getByTestId('operation-id').click();
```

Comment:
We should only use test ids as a last resort.

More context from https://testing-library.com/docs/queries/bytestid/:

> In the spirit of [the guiding principles](https://testing-library.com/docs/guiding-principles), it is recommended to use this only after the other queries don't work for your use case. Using data-testid attributes do not resemble how your software is used and should be avoided if possible. That said, they are way better than querying based on DOM structure or styling css class names.

So in this case it's better to use `modificationOperationEntry.getByRole('link')` (without name selector), because it resembles more how the user acts: looks at operations entry, clicks on the only link visible.